### PR TITLE
Base plotter service

### LINKED ISSUE #9: Build Base Plotter Service

### PR DESCRIPTION
**Pull request by antoniomf97**
Adds a basic plotter service, that receives an input (csv data file) and plot a simple graph. It takes into account basic configurations.

This PR closes #9.

### CODE REVIEW

File: services/.gitignore
Comment (easyg24) [inline services/.gitignore:1]: This shouldn't be specific of image, but all tmp files.

File: services/plotter/plotter.py
Comment (easyg24) [inline services/plotter/plotter.py:1]: For now self explanatory, but regard future comments when complexity increases.

**Review by easyg24 — CHANGES_REQUESTED** [review]
Please address changes.

File: services/.gitignore
Comment (antoniomf97) [reply services/.gitignore:1]: Changed.

**Review by easyg24 — APPROVED** [review]
Good!

**Pull request merged**